### PR TITLE
Add SSL config and File Storage methods

### PR DESCRIPTION
**Pull request by diogosilva0307**
Description
-----------

Test Steps
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/include/cellular_api.h [hunk 1]
```diff
@@ -639,6 +639,78 @@ CellularError_t Cellular_SocketRegisterClosedCallback( CellularHandle_t cellular
...
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] sslContextId The SSL context ID.
+ * @param[in] sslConfigurationParameter The SSL parameter to be configured.
+ * @param[in] inputArg The value to be passed to the SSL parameter.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_ConfigureSSLContext(CellularHandle_t cellularHandle,
```
Comment (lucid-at-dream) [inline source/include/cellular_api.h]: watch out for the formatting, in the previous declarations there's a blank space after the opening curved bracket. i.e. `( `

Comment (lucid-at-dream) [reply source/include/cellular_api.h]: Also, the arguments are column aligned

File: source/include/cellular_api.h [hunk 2]
```diff
@@ -639,6 +639,78 @@ CellularError_t Cellular_SocketRegisterClosedCallback( CellularHandle_t cellular
...
+/**
+ * @brief  Lists the information of a single file.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] filename Name of the file to be listed.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_VerifyFileFromStorage(CellularHandle_t cellularHandle,
```
Comment (lucid-at-dream) [inline source/include/cellular_api.h]: I can't understand what this function does after reading the docs and looking at its declaration. is it checking if the file exists? If so, change its name to "Cellular_CheckFileExists" perhaps. And update the documentation to be a bit more obvious

File: source/include/cellular_api.h [hunk 3]
```diff
@@ -639,6 +639,78 @@ CellularError_t Cellular_SocketRegisterClosedCallback( CellularHandle_t cellular
...
+/**
+ * @brief Delete a File in the Storage.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] filename Name of the file to be deleted.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_DeleteFileFromStorage(CellularHandle_t cellularHandle,
```
Comment (lucid-at-dream) [inline source/include/cellular_api.h]: I think you can drop the "FromStorage" from the function's name

**Review by lucid-at-dream — COMMENTED** [review]
I like it

**Review by lucid-at-dream — APPROVED** [review]

**Review by guildeol — APPROVED** [review]
Looks good to me :call_me_hand:

**Pull request merged**